### PR TITLE
CRUD for carriers

### LINKED ISSUE #7: CRUD for Carriers
Add the basic capability to support creating/reading/updating/and destroying a Carrier. Initial things we need to capture about a Carrier include:
* Item ID
* Name
* Manufacturer
* Model
* Color
* Size
* Location (association if Issue #5 is completed)
* Default Loan Length (standard = 30)
* ???

### PR DESCRIPTION
**Pull request by john-raber**
Resolves #7 

### Description
Guide questions:
  - What motivated this change (if not already described in an issue)?
     App needed a way to represent baby carriers
   
List any dependencies that are required for this change. (gems, js libraries, etc.)
Capybara

Include anything else we should know about. -->
All fields except size is required for the carrier
I think that some of the fields like item_id and location might need to be reworked

### Type of change
* New feature (non-breaking change which adds functionality)

### How Has This Been Tested?
Automated browser tests

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -15,6 +15,19 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "carriers", force: :cascade do |t|
+    t.integer "item_id"
```
Comment (cattywampus) [inline db/schema.rb]: This needs to be a `string` type because the IDs they generate include text characters, i.e. (B10A0901)

File: db/schema.rb [hunk 2]
```diff
@@ -15,6 +15,19 @@
...
+    t.string "name"
+    t.string "manufacturer"
+    t.string "model"
+    t.string "color"
+    t.integer "size"
+    t.integer "location_id"
+    t.integer "default_loan_length", default: 30
```
Comment (cattywampus) [inline db/schema.rb]: Let's denote the unit of measure here, being `days`

File: db/schema.rb (hunk 1 above)
Comment (john-raber) [reply db/schema.rb]: ids are now strings 👍

File: db/schema.rb (hunk 2 above)
Comment (john-raber) [reply db/schema.rb]: _days added for loan length

**Review by yarmiganosca — APPROVED** [review]

**Pull request merged**